docs/index.rst

```diff
@@ -7,21 +7,21 @@ Solidity
     :align: center
 
 Solidity is an object-oriented, high-level language for implementing smart
-contracts. Smart contracts are programs which govern the behaviour of accounts
+contracts. Smart contracts are programs that govern the behaviour of accounts
 within the Ethereum state.
 
-Solidity was influenced by C++, Python and JavaScript and is designed to target
+Solidity was influenced by C++, Python, and JavaScript and is designed to target
```

Comment:
We usually don't use the Oxford comma.

Reply:
It has been used in some cases. Its use isn't consistent so corrected it.

Reply:
Sure! I have the impression, though, that there should be more places where we do not sue it than there are where we us eit.